cosmoz-page-router.js

```diff
@@ -402,14 +407,15 @@
 					this._activateImport(route, url, eventDetail, importLink);
 				}
 			}
-		},
+		}
 
 		_hasCustomElement(elementName) {
-			const customElements = window.customElements;
-			return Polymer.telemetry.registrations.some(function (element) {
-				return element.is === elementName;
-			}) || customElements && customElements.get(elementName) != null;
-		},
+			let defined = false;
```

Comment:
Please use ```
return customElements.get(elementName) != null
```
`whenDefined` returns a `Promise` and defined will alows be false on return.

Reply:
Thanks.

I updated it, now I get a new error when testing against frontend:

```
Uncaught TypeError: Cannot read property 'routes' of undefined
    at HTMLElement._changeRoute (cosmoz-page-router.js:564)
    at HTMLElement._activateCustomElement (cosmoz-page-router.js:499)
    at HTMLElement._activateImport (cosmoz-page-router.js:431)
    at HTMLLinkElement.importLoadedCallback (cosmoz-page-router.js:369)
```

Reply:
But `_hasCustomElement` now returns `true`. :+1:

Reply:
Looks like `this.$.routes` is undefined.

Reply:
@megheaiulian Please have a look as discussed on Slack.